logic/commentLogic.js

```diff
@@ -13,12 +13,13 @@ module.exports = class CommentLogic {
       if (tipId === null || tipId === undefined || !text || !author || !signature || !challenge) {
         return res.status(400).send('Missing required field');
       }
-
-      const parentComment = typeof parentId !== 'undefined'
+      const parentComment = (typeof parentId !== 'undefined' && parentId !== '')
         ? await Comment.findOne({ where: { id: parentId } }, { raw: true }) : null;
-      if (parentComment === null && typeof parentId !== 'undefined') return res.status(400).send(`Could not find parent comment with id ${parentId}`);
+      if (parentComment === null && typeof parentId !== 'undefined' && parentId !== '') {
```

Comment:
can it be all, `null`, `undefined` or `''`? Otherwise let's only do the relevant checks

Reply:
it technically should not even be an empty string, its only a bug in the currently deployed frontend

Reply:
ok, then lets keep as is